hoomd/md/methods.py

```diff
@@ -454,14 +454,38 @@ def thermalize_thermostat_and_barostat_dof(self, seed):
             seed, self._simulation.timestep)
 
 
-class nph(NPT):
+class NPH(NPT):
     R""" NPH Integration via MTK barostat-thermostat..
 
     Args:
-        params: keyword arguments passed to :py:class:`NPT`.
-        gamma: (:py:obj:`float`, units of energy): Damping factor for the box degrees of freedom
+        filter (`hoomd.filter.ParticleFilter`): Subset of particles on which to
+            apply this method.
+
+        tau (`float`): Coupling constant for the thermostat (in time units).
```

Comment:
Remove tau from the docs, the argument list and the param dict. It relates to the thermostat.